pkg/infra/spot.go

```diff
@@ -377,7 +400,7 @@ func getMetadataMockDeployment(platform configv1.PlatformType) *appsv1.Deploymen
 					Containers: []corev1.Container{
 						{
 							Name:    "metadata-mock",
-							Image:   "golang:1.14",
+							Image:   "quay.io/openshifttest/golang@sha256:8f1c43387f0a107535906c7ee918a9d46079cc7be5e80a18424e8558d8afc702",
```

Comment:
What is this image? Could we add a comment with the human friendly name so we know how to update this in the future

Reply:
This is golang image, mirrored in quay.io/openshifttest, so that disconnected cluster also can access. I will add a comment.

Reply:
Can you include in the comment what tag the sha represents? Is this go 1.23 for example?

Reply:
will update